### ISSUE TITLE
Joining forces

### ISSUE THREAD

**Issue by ecarreras** [flabe81/pyCadastro#1]
What about joining forces? We already have a working version https://github.com/gisce/pycatastro

**Comment by fabiore** [comment]
Yes! but you have to explain me how to join the forces !Let me know!

**Comment by fabiore** [comment]
Next month we are going to work on this, than if we want t join our force we need a little bit of coordination.Can you coordinate  or give me a small speech on how to ?
